MovieQuiz/Presentation/MovieQuizPresenter.swift

```diff
@@ -0,0 +1,127 @@
+import Foundation
+import UIKit
+
+final class MovieQuizPresenter: QuestionFactoryDelegate {
+    private let statisticService: StatisticService!
```

Comment:
здесь нет смысла использовать неявное разворачивание(implicitly unwrapping) потому что в инициализаторе  гарантировано будет установлено значение 
```suggestion
    private let statisticService: StatisticService
```

Во вьюконтроллере так было нельзя, так как присваивание происходило уже после инициализации объекта (во viewDidLoad)

С фабрикой тоже самое